diagnostic.md

```diff
@@ -34,5 +35,5 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{my-contact/my-phone number=number}}
```

Comment:
You've got the basic idea, but you need to iterate through the contacts.
```
{{#each contact.phones as |phone|}} <!-- correct object property access -->
   {{my-phone phone=phone}} <!-- sub-component included, binding necessary -->
{{/each}}
```

.5/1